pkgs/by-name/op/opencode/package.nix

```
@@ -71,7 +73,7 @@ stdenvNoCC.mkDerivation (finalAttrs: {
     nativeBuildInputs = [
       bun
       writableTmpDirAsHomeHook
-    ];
+    ] ++ lib.optional installShellCompletions installShellFiles;
```

Sry, my mistake these are the wrong `nativeBuildInputs`. You need to add this to line [116](https://github.com/MXfive/nixpkgs/blob/687e2d9ac086aff25788137f0cdece094467a86d/pkgs/by-name/op/opencode/package.nix#L116).

```suggestion
    ];
```